atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowWriteTable.java

```diff
@@ -231,6 +288,50 @@ private void deleteOverflow(String overflowTable, List<Object[]> args) {
                 args);
     }
 
+    private void deleteOverflowRange(String overflowTable, String shortTableName, RangeRequest range) {
```

Comment:
This currently has no testing, consider adding some to the internal oracle testing respository - dbkvs-oracle-driver